styles/components/profil.css

```diff
@@ -0,0 +1,39 @@
+.profil {
+  margin: 0;
```

Comment:
I saw this margin already quite often - maybe an idea to summarize it in utilities? 😊

Reply:
yes this is a good idea 👍🏻